docs/introduction/index.rst

```diff
@@ -19,7 +19,13 @@ Features
 --------
 
 Renku consists of a collection of services, including a web-based user interface
-and a command-line client.
+and a command-line client, exploiting in a coherent setup the joint features of:
+
+* GitLab -  repository management
+* JupyterHub - Interactive Notebooks
+* Kubernetes - Container Orchestration
+* Keycloak - Identity and Access Management
+* Python - Scripting & glue logic
```

Comment:
hmm I don't know about including python here -- it's not really a technology. It's clearly different than the other things on the list so I would remove it. Maybe also add Common Workflow Language and hyperlinks to the web pages for these components?